src/summary/documentation/dimensions-card-level-two.test.tsx

```diff
@@ -71,7 +71,9 @@ it("renders dates according to designs", () => {
   );
   expect(formatDate(sometimeThisYear)).toBe(`26-Apr 06:49 AM`);
   expect(formatDate(today.toString())).toBe(
-    `Today ${get12Hour(today.getHours())}:${zeroBase(today.getMinutes())} PM`
+    `Today ${get12Hour(today.getHours())}:${zeroBase(today.getMinutes())} ${
+      today.getHours() < 12 ? "A" : "P"
```

Comment:
This test would only work if run in the morning 😄

Reply:
`A` or `AM`?

Reply:
oops saw the `M`

Reply:
The `M` is on the next line (line 76) - it's there for both AM and PM which is why it's not in the ternary